.github/workflows/e2e-hotel.yml

```diff
@@ -0,0 +1,151 @@
+name: Hotel-App End-to-End Tests
+
+on:
+  workflow_dispatch:
+#     schedule:
+#       - cron: "0 9 * * 1"
+  push:
+    branches: [main]
+    paths:
+      - "benchmarks/hotel-app/**"
+      - "utils/**"
+      - "tools/**"
+      - "runner/**"
+
+  pull_request:
+    branches: [main]
+    paths:
+      - "benchmarks/hotel-app/**"
+      - "utils/**"
+      - "tools/**"
+      - "runner/**"
+
+env:
+  GOOS: linux
+  GO111MODULE: on
+  PORT: 50051
+  KUBECONFIG: /etc/kubernetes/admin.conf
+
+jobs:
+  build-and-push:
+    name: Build and push all images
+    runs-on: ubuntu-20.04
+    strategy:
```

Comment:
We should remove this since it doesn't make sense in a context without matrix (I have removed them in auth, fibonacci, online shop atleast).